.devcontainer/setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+env_variables=$(yq eval '.env | to_entries | .[] | .key + "=" + .value' .github/workflows/end2end.yaml | sed 's/\${{[^}]*}}//g') && export $env_variables
+export GIT_ACCESS_TOKEN=${GITHUB_TOKEN}
```

Comment:
This doesn't export `ZENKO_MONGODB_SHARDED` which is why mongo was not deployed in sharded mode the other day.
Some env vars like `ZENKO_MONGODB_SHARDED` and others are not defined in the env section, maybe we should add them.

Reply:
Oh never mind you manually added it to `containerEnv`